turbo.json

```diff
@@ -3,14 +3,23 @@
   "pipeline": {
     "build": {
       "outputs": ["build/**", ".next/**", "!.next/cache/**"],
-      "dependsOn": ["^build"]
+      "dependsOn": ["generate", "^build"]
     },
     "lint": {},
     "lint:fix": {},
     "dev": { "dependsOn": ["build"] },
     "start": { "dependsOn": ["build"] },
     "clean": { "cache": false },
     "test": { "dependsOn": ["build"] },
-    "analyze": {}
+    "analyze": {},
+    "generate": {
+      "outputs": [
+        "src/codegen/**",
+        "config/wallet-registry.ts",
+        "config/wallet-assets.ts",
+        "config/chain-infos.ts"
+      ],
+      "dependsOn": ["^generate"]
+    }
```

Comment:
Call me nitpicky, but could we sort these `pipeline` members in rough order of execution? Would prob be a bit more readable for new devs.